bridge-impl/src/main/java/com/liferay/faces/bridge/renderkit/html_basic/internal/ResponseWriterBridgeBodyImpl.java

```diff
@@ -37,6 +38,7 @@
 	private static final Logger logger = LoggerFactory.getLogger(ResponseWriterBridgeBodyImpl.class);
 
 	// Private Data Members
+	private Stack<String> elementsStack = new Stack<String>();
```

Comment:
@ngriffin7a, that's a good point. The stack is overkill and `currentElement` should work fine instead.